src/fisher.jl

```diff
@@ -137,7 +137,9 @@ end
 
 function pvalue_both_minlike(x::FisherExactTest, ω::Float64=1.0)
     a, b, c, d = reorder(x.a, x.b, x.c, x.d)
-
+    if a == c == 0 || b == d == 0
+        return 1
```

Comment:
```suggestion
        return 1.0
```
For type stability. Looking through the code, I think everything for Fisher's exact test ends up as `Float64`, so we don't need to do any shenanigans to figure out the appropriate return type.